### ISSUE TITLE
Add info action

### ISSUE THREAD

**Issue by Macainian** [Macainian/Django-Maced#3]
Currently this feature is always disabled. This is because it hasn't been implemented yet. This will take a bit of work to make it do what I REALLY want it to do.

The short explanation is to allow it to show stats about the item in the database as well as the creation info of the item selected. This will take some serious work to add this, but a smaller version will be easy to add.

The smaller version will simply add the same stuff as every other action does and will not be editable. This will be useful when permissions come into play since a user may want to see what it has while still not being able to edit/delete/merge it.

Once the smaller version is complete, this will be opened as a new issue and be written in terms of the full version (or at least the next step to it).
